### ISSUE TITLE
0.3.2 SequentialLayout is missing opacity from parentSpec in returned render spec

### ISSUE THREAD

**Issue by SpaceJoe** [Famous/famous#530]
I'd say this:

```js
        return {
            transform: parentSpec.transform,
            origin: parentSpec.origin,
            size: this.getSize(),
            target: result
        };
```

should be:

```js
        return {
            opacity: parentSpec.opacity,
            transform: parentSpec.transform,
            origin: parentSpec.origin,
            size: this.getSize(),
            target: result
        };
```

**Comment by michaelobriena** [comment]
https://github.com/Famous/famous/pull/561

**Comment by TheAlphaNerd** [comment]
fixed by 13a0298
